src/tests/uri_tests.cxx

```diff
@@ -1,138 +1,188 @@
+#include <gmock/gmock-actions.h>
 #include "../uri.h"
-#include <gtest/gtest.h>
```

Comment:
Why do you include `gmock-actions.h` instead of `gtest.h`?

Reply:
not sure, will revert back to using gtest. CLion must have been playing up 😄